simplified-books-database/src/main/java/org/nypl/simplified/books/book_database/BookDatabase.kt

```diff
@@ -113,8 +128,8 @@ class BookDatabase private constructor(
           Book(
             id = id,
             account = this.owner,
-            cover = null,
-            thumbnail = null,
+            cover = cover,
```

Comment:
This ensures we don't blow away the cover and thumbnail information when updating a stored entry. I'm unsure if we currently do this anywhere, but it was possible.